cot_probing/attn_probes_data_proc.py

```diff
@@ -115,41 +115,65 @@ def split_data(
 ) -> tuple[
     tuple[DataLoader, DataLoader, DataLoader], tuple[list[int], list[int], list[int]]
 ]:
-    data_seed = data_loading_kwargs["data_seed"]
-    assert isinstance(data_seed, int)
-    test_split = data_loading_kwargs["test_split"]
-    assert isinstance(test_split, float)
-    validation_split = data_loading_kwargs["validation_split"]
-    assert isinstance(validation_split, float)
+    """Split data into train/val/test sets using cross-validation folds"""
+    train_seed = data_loading_kwargs["train_seed"]
+    assert isinstance(train_seed, int)
     batch_size = data_loading_kwargs["batch_size"]
     assert isinstance(batch_size, int)
-    setup_determinism(data_seed)
+
+    # Get fold information
+    fold = data_loading_kwargs.get("fold", 0)
+    n_folds = data_loading_kwargs.get("n_folds", 1)
+    assert isinstance(fold, int) and isinstance(n_folds, int)
+    assert 0 <= fold < n_folds
+
+    setup_determinism(train_seed)
```

Comment:
This should be cv seed?

Reply:
I thought we were going to use the train seed for shuffling the data after the fold split. The cv is used for shuffling before the fold split.